### PR TITLE
document how we use migrations nginx gunicorn supervisord and postgresql

### PR DESCRIPTION
**Pull request by g1039**

### CODE REVIEW

**Comment by g1039** [comment]
@miltontony Please review

File: docs/platforms.rst [hunk 1]
```diff
@@ -32,10 +32,10 @@ We deploy Django in the following stack:
 
 - Ubuntu Server (current LTS release)
 - haproxy_ for load balancing where appropriate
-- nginx_
-- gunicorn_
-- supervisord_
-- postgresql_
+- nginx_ Web server that we use to run our Django applications in production.
+- gunicorn_ is the application server we're using to servce our Django applications with
```
Comment (miltontony) [inline docs/platforms.rst]: typo on `serve`

File: docs/tools.rst [hunk 2]
```diff
@@ -182,7 +182,7 @@ Django Applications
 For Django applications, some applications are mandatory:
 
 1. Sentry_ for application reporting.
-2. South_ for managing database schema changes.
+2. Migrations_ for handling database schema changes
```
Comment (miltontony) [inline docs/tools.rst]: Let's call this `Django Migrations`

**Comment by g1039** [comment]
@miltontony Ready for re-review

**Comment by miltontony** [comment]
👍

**Pull request merged**